requirements/edx/paver.in

```diff
@@ -12,7 +12,7 @@
 
 edx-opaque-keys                     # Create and introspect course and xblock identities
 lazy                                # Lazily-evaluated attributes for Python objects
-libsass==0.10.0                     # Python bindings for the LibSass CSS compiler
+libsass                             # Python bindings for the LibSass CSS compiler
```

Comment:
Constraint is moved over to `constraints.txt` so that it can be shared by `assets.txt`.